chpl/chpl-api/e2e/collections/products-controller.postman_collection.json

```diff
@@ -0,0 +1,55 @@
+{
+	"info": {
+		"_postman_id": "6e1c03a3-35b0-496f-99c3-30c89f93aaa3",
+		"name": "products-controller",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "GET /products",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "e517dbed-0a00-403e-8fe6-429adda650cc",
+						"exec": [
+							"pm.test(\"Invalid api key parameter should return 401 status\", function () {\r",
+							"    pm.response.to.have.status(401);\r",
```

Comment:
Is there any reason some controllers have both valid and invalid api test and some dont. Why not add valid api key test?

Reply:
fixed